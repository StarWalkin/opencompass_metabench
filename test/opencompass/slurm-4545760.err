  0%|          | 0/3 [00:00<?, ?it/s]                                       0%|          | 0/3 [00:00<?, ?it/s]slurmstepd: error: *** JOB 4545760 ON gpua039 CANCELLED AT 2024-08-18T20:57:50 ***
