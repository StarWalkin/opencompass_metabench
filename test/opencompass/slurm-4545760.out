/scratch/bcwg/Metabench/anaconda/envs/opencompass/bin/python3
08/18 16:01:32 - OpenCompass - INFO - Loading math_gen: configs/datasets/math/math_gen.py
08/18 16:01:32 - OpenCompass - INFO - Loading hf_gemma2_9b_it: configs/models/gemma/hf_gemma2_9b_it.py
08/18 16:01:32 - OpenCompass - INFO - Loading hf_glm4_9b_chat: configs/models/chatglm/hf_glm4_9b_chat.py
08/18 16:01:32 - OpenCompass - INFO - Loading hf_phi_3_small_8k_instruct: configs/models/phi/hf_phi_3_small_8k_instruct.py
08/18 16:01:32 - OpenCompass - INFO - Loading example: configs/summarizers/example.py
08/18 16:01:32 - OpenCompass - INFO - Current exp folder: outputs/default/20240818_160132
08/18 16:01:32 - OpenCompass - WARNING - SlurmRunner is not used, so the partition argument is ignored.
08/18 16:01:32 - OpenCompass - INFO - Partitioned into 3 tasks.
